src/client/components/equalizer/equalizer.js

```diff
@@ -219,16 +224,21 @@ class Equalizer extends Component {
 
     return (
       <div className="graphicEqualizer">
-        <Graphicequaliser
-          width={widthCanvas}
-          height={heightCanvas}
-          getCanvasEl={setCanvasToState}
-        />
+        <div style={{ display: display2 }}>
```

Comment:
Those ternarys could be made here, no need to create variables for that

Reply:
@tonypntsrn done